.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: php
+php:
+#  - '5.4' -- breaks for some weird reason (https://travis-ci.org/wikimedia/popularpages/jobs/216541216)
```

Comment:
Is this a phpunit incompatibility? You might have to downgrade phpunit to ^4.8 (the last version for php >=5.3.3).